components/Locale/basic_usage.rst

```diff
@@ -0,0 +1,49 @@
+Basic Usage
+===========
+
+LocaleContext
+-------------
+
+The **LocaleContext** allows you to manage the currently used locale.
+
+.. code-block:: php
+
+	<?php
+
+	use Sylius\Component\Locale\Context\LocaleContext;
+
+	$storage; //Your's storage
+
+	$localeContext = new LocaleContext($storage, 'en');
+
+	$localeContext->getDefaultLocale() //Output will be 'en'.
+	$localeContext->getCurrentLocale() //Output based on your storage implementation.
+	$localeContext->setCurrentLocale('us') //It will set your default locale in your storage.
+
+.. note::
+	For more detailed information go to `Sylius API LocaleContext`_.
+
+.. _Sylius API LocaleContext: http://api.sylius.org/Sylius/Component/Locale/Context/LocaleContext.html
+
+LocaleProvider
+--------------
+
+The **LocaleProvider** allows you to get all available locales.
+
+.. code-block:: php
+
+	<?php
+
+	use Sylius\Component\Locale\Provider\LocaleProvider;
+
+	$localeRepository; \\Repository model
```

Comment:
Yeah, these comments are not that clear, I'd say we use InMemoryRepository in example. Then we just need to implement it soon. :D